cmd/bucket-handlers.go

```diff
@@ -601,10 +601,10 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 	}
 
 	if objectAPI.IsEncryptionSupported() {
-		if hasSSECustomerHeader(formValues) && !hasSuffix(object, slashSeparator) { // handle SSE-C requests
+		if ContainsSSECustomerHeader(formValues) && !hasSuffix(object, slashSeparator) { // handle SSE-C requests
```

Comment:
Isn't better to not capitalize the first character here ? to avoid reader's mind questions. I guess this function won't be exported anyway.

Reply:
Hm, I don't think so. If you think of somebody taking a look `/cmd` at e.g. https://godoc.org/github.com/minio/minio - there is no meaning to that doc. `cmd` "randomly" exports stuff.
If we don't follow a package-style conceptually for development I don't think we can argue that our code should follow a package-style... 

What I try to do is to make functions which should be used by "clients" - like another layer (in this case S3-layer) exported and low-level functions un-exported.
To give a concrete example:
 - [Intended for "outer" usage](https://github.com/minio/minio/pull/5802/files#diff-737914b14614fb9579da73077874ce5dR228)
 - [Intended for "inner" usage](https://github.com/minio/minio/pull/5802/files#diff-737914b14614fb9579da73077874ce5dR185)

I don't think that exported vs. un-exported is something we can rely on using our current code architecture.

Reply:
We could certainly avoid exporting since cmd functions are not meant for that.

Reply:
Well but what is the point of restricting ourself to not export something if `cmd` is not meant to be imported?
The concept exportable / not-exportable only makes sense **if** `cmd` is a package that may be imported.

Reply:
If this is our goal we can simply move cmd to internal/cmd and the problem is gone for all functions...